python/mxnet/test_utils.py

```diff
@@ -114,6 +110,8 @@ def rand_ndarray(shape, stype, density=None):
 def rand_shape_2d(dim0=10, dim1=10):
     return rnd.randint(1, dim0), rnd.randint(1, dim1)
 
+def rand_shape_3d(dim0=10, dim1=10, dim2=10):
+    return rnd.randint(1, dim0), rnd.randint(1, dim1), rnd.randint(1, dim2)
```

Comment:
Since the 'high' value of randint is exclusive (see [here](https://docs.scipy.org/doc/numpy/reference/generated/numpy.random.randint.html)) I'd suggest to write `rnd.randint(1, dim0+1)`, etc., so that passing dim0=1 returns one-dimensional shape instead of an error.